### ISSUE TITLE
PHP code is off

### ISSUE THREAD

**Issue by Lvl4Sword** [dustball/secretchat.site#1]
```
<?php

if (!$_SERVER['HTTPS']) {
  header('Location: https://secretchat.site/');
  exit;
}
```
You should just force HTTPS on the site and use a sameorigin header and it'd be better.

**Comment by dustball** [comment]
Do you mean like this:

<Location />
RewriteEngine On
RewriteCond %{HTTPS} off
RewriteRule (.*) https://%{HTTP_HOST}%{REQUEST_URI}
</Location>

**Comment by Lvl4Sword** [comment]
I meant just enable port 443 and don't even have an option for 80.
That's also not setting a sameorigin or nosniff header ( which both should be set )

**Comment by dustball** [comment]
HTTP needs to be enabled so that people don't get a connection refused message when typing "secretchat.site" into their browser's Location bar.

Add these lines of code?

header('X-Frame-Options: SAMEORIGIN');
header('X-Content-Type-Options: nosniff');

I want to do this in PHP, and not in Apache config, to make these settings portable.  People can simply deploy the PHP code and it just works.

**Comment by dustball** [comment]
Resolved 045d679
